scripts/cdist

```diff
@@ -138,19 +140,11 @@ def commandline():
         # created in it.
         if args_dict['out_path'] is None:
             args.out_path = tempfile.mkdtemp()
-            is_temp_dir = True
-        else:
-            is_temp_dir = False
         # if remote-exec and/or remote-copy args are None then user
         # didn't specify command line options nor env vars:
         # inspect multiplexing options for default cdist.REMOTE_COPY/EXEC
         if args_dict['remote_copy'] is None or args_dict['remote_exec'] is None:
             control_path_dir = args.out_path
-            # only rmtree if it is temp directory;
-            # if user specifies out_path do not remove it
-            if is_temp_dir:
-                import atexit
-                atexit.register(lambda: shutil.rmtree(control_path_dir))
```

Comment:
atexit rmtree is not needed. Saving cache removes out_path already and this removed code would throw error that path does not exist.